modules/edge-voting/src/voting.rs

```diff
@@ -139,35 +138,45 @@ decl_module! {
 		/// A function that reveals a vote commitment or serves as the general vote function.
 		///
 		/// There are currently no cryptoeconomic incentives for revealing commited votes.
-		pub fn reveal(origin, vote_id: u64, vote: VoteOutcome, secret: Option<VoteOutcome>) -> Result {
+		pub fn reveal(origin, vote_id: u64, vote: Vec<VoteOutcome>, secret: Option<VoteOutcome>) -> Result {
 			let _sender = ensure_signed(origin)?;
 			let mut record = <VoteRecords<T>>::get(vote_id).ok_or("Vote record does not exist")?;
 			ensure!(record.data.stage == VoteStage::Voting, "Vote is not in voting stage");
-			// Check vote is for a valid outcome
-			ensure!(record.outcomes.iter().any(|o| o == &vote), "Vote outcome is not valid");
+			// Check vote is for valid outcomes
+			for i in 0..vote.len() {
```

Comment:
What happens if I vote for multiple items in a non-ranked choice vote?

Reply:
Addressed